android/app/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+
+    defaultConfig {
+        applicationId "com.sample"
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+}
+
+def hermesPath = "node_modules/hermes-engine/android"
+
+dependencies {
+    debugImplementation files("$rootDir/../$hermesPath/hermes-debug.aar")
+    releaseImplementation files("$rootDir/../$hermesPath/hermes-release.aar")
+
+    implementation "com.facebook.react:react-native:0.61.5"
```

Comment:
Can we use 0.60.5 instead? We're not quite on 0.61 yet.

Reply:
Yes. In fact, we should probably avoid depending on particular version of RN in gradle. Will fix it.